core/src/main/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerService.java

```diff
@@ -70,10 +70,18 @@
     public static final Setting<CircuitBreaker.Type> IN_FLIGHT_REQUESTS_CIRCUIT_BREAKER_TYPE_SETTING =
         new Setting<>("network.breaker.inflight_requests.type", "memory", CircuitBreaker.Type::parseValue, Property.NodeScope);
 
+    public static final Setting<ByteSizeValue> ACCOUNTING_CIRCUIT_BREAKER_LIMIT_SETTING =
+        Setting.memorySizeSetting("network.breaker.accounting.limit", "100%", Property.Dynamic, Property.NodeScope);
+    public static final Setting<Double> ACCOUNTING_CIRCUIT_BREAKER_OVERHEAD_SETTING =
+        Setting.doubleSetting("network.breaker.accounting.overhead", 1.0d, 0.0d, Property.Dynamic, Property.NodeScope);
+    public static final Setting<CircuitBreaker.Type> ACCOUNTING_CIRCUIT_BREAKER_TYPE_SETTING =
+        new Setting<>("network.breaker.accounting.type", "memory", CircuitBreaker.Type::parseValue, Property.NodeScope);
```

Comment:
It seems slightly weird that this setting is prefixed with `network` but I guess that because all the breakers have this prefix?

Reply:
Whoops you're right, this should be `indices` instead of `network`. This is a typo from copying the other settings. I'll fix this, thanks!